src/modules/commands/discord/urbandictionary.ts

```diff
@@ -28,9 +28,47 @@ export default new Command<discord.Message>({
     lastArgNumber: 1,
   },
   callback: async ({ args, reply, env, raw }) => {
+    async function removeReaction(reaction: discord.MessageReaction) {
+      try {
+        await reaction.users.remove(raw.author)
+      } catch (error) {
+        console.log('Error removing reaction:', error)
+      }
+    }
+
+    function genDefinitionEmbed(
+      definition: Definition,
+      len: number,
+    ): discord.MessageEmbed {
+      const definitionEmbed = new discord.MessageEmbed()
+        .setColor('#647CC4')
+        .setTitle(`${definition.word} (N°${defN + 1}/${len})`)
+        .setURL(definition.permalink)
+        .setAuthor('Urban dictionary', '', 'https://www.urbandictionary.com')
+        .setThumbnail('https://i.imgur.com/A6nvY85.png')
+        .addFields(
+          {
+            name: 'Definition',
+            value: definition.definition.replace(/\[|\]/g, '*'),
+          },
+          { name: 'Example', value: definition.example.replace(/\[|\]/g, '*') },
+        )
+        .addField('Written on', definition.writtenOn.substring(0, 10), true)
+        .addField('Thumbs up', definition.thumbsUp, true)
+        .addField('Thumbs down', definition.thumbsDown, true)
+        .setFooter('Interactive')
+        .setTimestamp()
+
+      return definitionEmbed
+    }
+
+    function mod(n: number, m: number): number {
```

Comment:
Please put this in utils/math.ts and change it from `function mod ...` to `const mod = ...` if you would be so kind.